pyhiveapi/apyhiveapi/api/hive_api.py

```diff
@@ -210,11 +266,13 @@ def getProducts(self):
 
     def getActions(self):
         """Call the get actions endpoint."""
-        url = self.urls["base"] + self.urls["actions"]
+        url = self.urls["all"]
```

Comment:
How come this has been updated to point to all? getDevices & getProducts still points to the individual endpoints. if we can get all data from the all endpoint maybe we should remove the individual endpoints. What do you think?

Reply:
I have no idea, I did this in July.  Sorry.